_use-case-gallery/external-recommender/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: External recommender
+subheadline: Example project
+permalink: /use-case-gallery/external-recommender/
+# screenshot: screenshot.png
+# thumbnail: screenshot-thumb.png
+example-project: External+Recommender+Example+Project_2018-12-07_1608.zip
+---
+
+This example provides a ready-to-use project show-casing the use of an *External Recommender* in INCEpTION for annotating named entities
+and part-of-speech tags. Recommenders provide interactive annotation support by suggesting
+potential labels.
+
+External recommenders are web services that expose a machine learning classifier over a HTTP web service.
+They can predict annotations for given documents and optionally be trained on new data. The external recommender
+support in INCEpTION allows to e.g. integrate custom classifiers written in a different programming language
+or to use an already pre-trained model.
+
+The protocol used by INCEpTION to communitcate with an external recommender is documented in the [INCEpTION Developer Guide](https://zoidberg.ukp.informatik.tu-darmstadt.de/jenkins/job/INCEpTION%20(GitHub)%20(master)/de.tudarmstadt.ukp.inception.app$inception-app-webapp/doclinks/3/#_external_recommender_api_overview).
+
+To get started with external recommenders, we provide several external recommender implementations that 
+can serve as examples and as a basis for custom implementations. They are described in the
```

Comment:
If I understand correctly, there is only the spacy recommender, right? So, maybe re-formulate to:

...for custom implementations. In this example, we use the spaCy external recommender that is described in the following. The link to the GitHub repository contains documentation and an installation guide.